.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: trusty
```

Comment:
Not sure why I missed this before. You also have to change the `sudo` instruction: https://docs.travis-ci.com/user/trusty-ci-environment#Using-the-Trusty-beta

```
sudo: required
dist: trusty
```